### PR TITLE
chore: add volunteer  Saurav-Pant

### PR DESCRIPTION
**Pull request by Saurav-Pant**
I want to be a part of AskDevs as a volunteer.

### CODE REVIEW

File: data/users/Saurav-Pant.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Saurav Pant",
+    "bio": "FrontEnd Developer",
```
Comment (codewithvoid) [inline data/users/Saurav-Pant.json:3]: fix casing. `Frontend Developer`

File: data/users/Saurav-Pant.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "image": "https://avatars.githubusercontent.com/u/103263909?s=96&v=4",
+    "categories": [
+      {
+        "slug": "web-development",
+        "topics": ["reactjs", "css", "javascript"]
```
Comment (codewithvoid) [inline data/users/Saurav-Pant.json:8]: can you mention javascript in separate programming and removing it from here. E.g

```json
{
 "slug" : "programming",
"topics": ["JavaScript"]
}
```

File: data/users/Saurav-Pant.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Saurav Pant",
+    "bio": "FrontEnd Developer",
+    "image": "https://avatars.githubusercontent.com/u/103263909?s=96&v=4",
```
Comment (codewithvoid) [inline data/users/Saurav-Pant.json:4]: use simpler url https://avatars.githubusercontent.com/u/103263909?v=4

**Review by codewithvoid — CHANGES_REQUESTED** [review]

File: data/users/Saurav-Pant.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Saurav Pant",
+    "bio": "Frontend Developer",
+    "image": "https://avatars.githubusercontent.com/u/103263909?v=4",
+    "categories": [
+      {
+        "slug": "web-development",
```
Comment (codewithvoid) [inline data/users/Saurav-Pant.json:7]: @Saurav-Pant change this to programming

Comment (Saurav-Pant) [reply data/users/Saurav-Pant.json:7]: @codewithvoid  can you plz tell me Why do I write programming here, is javascript not in web-development categories??

Comment (codewithvoid) [reply data/users/Saurav-Pant.json]: Thank you!

**Review by codewithvoid — APPROVED** [review]

**Pull request merged**